### ISSUE TITLE
Guard crashes (Errno::ENOENT) when trying to connect my browser page to it.

### ISSUE THREAD

**Issue by Veske** [guard/guard-livereload#148]
So I made a fresh install of the latest guard gem and tried to use it.

When I launch the webserver and guard and navigate to the page and press the guard extension button, I see this in my guard logs:
```
sponzme  bundle exec guard -d
00:20:09 - DEBUG - Notiffany: gntp not available (Please add "gem 'ruby_gntp'" to your Gemfile and run your app with "bundle exec".).
00:20:09 - DEBUG - Notiffany: growl not available (Unsupported platform "linux-gnu").
00:20:09 - DEBUG - Notiffany: terminal_notifier not available (Unsupported platform "linux-gnu").
00:20:09 - DEBUG - Notiffany: libnotify not available (Please add "gem 'libnotify'" to your Gemfile and run your app with "bundle exec".).
00:20:09 - DEBUG - Command execution: which notify-send
00:20:09 - DEBUG - Command execution: {"ALTERNATE_EDITOR"=>"false"} emacsclient --eval '1'
00:20:09 - DEBUG - Notiffany: emacs not available (Emacs client failed).
00:20:09 - DEBUG - Notiffany: tmux not available (:tmux notifier is only available inside a TMux session.).
00:20:09 - DEBUG - Notiffany: file not available (No :path option given).
00:20:09 - DEBUG - Command execution: which notify-send
00:20:09 - DEBUG - Notiffany is using NotifySend to send notifications.
00:20:09 - DEBUG - Notiffany is using TerminalTitle to send notifications.
00:20:10 - DEBUG - Command execution: hash stty
00:20:10 - DEBUG - Guard starts all plugins
00:20:10 - DEBUG - Hook :start_begin executed for Guard::LiveReload
00:20:10 - DEBUG - Hook :start_end executed for Guard::LiveReload
00:20:10 - INFO - LiveReload is waiting for a browser to connect.
00:20:10 - INFO - Guard is now watching at '/home/kaspar/Documents/Coding-projects/RubymineProjects/sponzme'
00:20:10 - DEBUG - Start interactor
[1] guard(main)> 00:20:16 - INFO - Browser connected.
00:20:16 - DEBUG - Serving file /tmp/livereload.js20151026-12210-j3r6li
00:20:16 - DEBUG - Interactor was stopped or killed
/home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/guard-livereload-2.5.0/lib/guard/livereload/websocket.rb:38:in `size': No such file or directory @ rb_file_s_size - /tmp/livereload.js20151026-12210-j3r6li (Errno::ENOENT)
        from /home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/guard-livereload-2.5.0/lib/guard/livereload/websocket.rb:38:in `_serve_file'
        from /home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/guard-livereload-2.5.0/lib/guard/livereload/websocket.rb:60:in `_serve'
        from /home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/guard-livereload-2.5.0/lib/guard/livereload/websocket.rb:23:in `dispatch'
        from /home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/em-websocket-0.5.1/lib/em-websocket/connection.rb:78:in `receive_data'
        from /home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/eventmachine-1.0.8/lib/eventmachine.rb:193:in `run_machine'
        from /home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/eventmachine-1.0.8/lib/eventmachine.rb:193:in `run'
        from /home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/guard-livereload-2.5.0/lib/guard/livereload/reactor.rb:50:in `_start_reactor'
        from /home/kaspar/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/guard-livereload-2.5.0/lib/guard/livereload/reactor.rb:11:in `block in initialize'

```

What could be the problem here?

**Comment by karic** [comment]
Please add "gem 'ruby_gntp'" and "gem 'libnotify'" to your Gemfile and
run your app with "bundle exec".


I am new here so that is what I have seen from the logs, try that and
come back. If it doesn't solve it maybe someone else will have few
ideas.

**Comment by Veske** [comment]
Problem seems to persist with the same error.

**Comment by e2** [comment]
For some reason the temporary file becomes unavailable... I wonder why ...

**Comment by e2** [comment]
@Veske - can you try this branch in your `Gemfile`:

```ruby
gem 'guard-livereload', github: 'guard/guard-livereload', branch: 'tmpfile_workaround'
```

I think it should keep the temporary file long enough to stream it ...

**Comment by e2** [comment]
If anyone else has this error - let me know if the branch fixes it or not. 

If it does, I'll make a quick bugfix release.

Thanks.

**Comment by Veske** [comment]
@e2 This branch seemed to fix the problem for me. Thank you!

**Comment by e2** [comment]
Fixed in guard-livereload 2.5.1
